core/src/main/scala/flatgraph/Graph.scala

```diff
@@ -143,4 +144,33 @@ class Graph(val schema: Schema, val storagePathMaybe: Option[Path] = None) exten
     }
   }
 
+  def cloneDataFrom(other: Graph): this.type = {
```

Comment:
I think you want to remove this? (only one API, and copy is fine)

Reply:
oh, yes of course, thanks for catching :)